llama-index-integrations/storage/kvstore/llama-index-storage-kvstore-azure/llama_index/storage/kvstore/azure/base.py

```diff
@@ -212,9 +213,10 @@ async def aput(
         table_name = (
             DEFAULT_COLLECTION if not collection else sanitize_table_name(collection)
         )
-        await self._atable_service_client.create_table_if_not_exists(
-            table_name
-        ).upsert_entity(
+        atable_service_client = (
```

Comment:
we should call this `atable_client`

Reply:
Thanks for reviewing @nerdai 
I disagree. Those are two different concepts/objects. Table client vs table service client.

Reply:
Right. Unless I read this resource incorrectly:

https://learn.microsoft.com/en-us/python/api/overview/azure/data-tables-readme?view=azure-python#creating-a-table

The table_service_client allows us to create a table and returns a table_client

Reply:
Normalized the naming.